changelogs/fragments/sanity_fix_modules.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- atomic_container - fix sanity check.
```

Comment:
I think it doesn't need a changelog fragment since it's mostly documentation changes. The only functional change is `elements='str'`, though I think it's ok to not explicitly mention it.

Reply:
oK.